pkg/products/amqonline/reconciler.go

```diff
@@ -350,3 +359,63 @@ func (r *Reconciler) reconcileBlackboxTargets(ctx context.Context, installation
 
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
+
+func (r *Reconciler) reconcilePrometheusRule(ctx context.Context, installation *integreatlyv1alpha1.Installation, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	monitoringConfig := config.NewMonitoring(config.ProductConfig{})
+	keycloakServicePortCount := "2"
+	rule := &monitoringv1.PrometheusRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rhmi-amq-online-sli",
+			Namespace: r.Config.GetNamespace(),
+		},
+	}
+
+	rules := []monitoringv1.Rule{
+		{
+			Alert: fmt.Sprintf("AMQOnlineConsoleUnavailable"),
+			Annotations: map[string]string{
+				"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md",
+				"message": fmt.Sprintf("AMQ-SLI-1.1: AMQ Online console is not available in namespace \"%s\"", r.Config.GetNamespace()),
+			},
+			Expr:   intstr.FromString(fmt.Sprintf("absent(kube_endpoint_address_available{endpoint=\"console\",namespace=\"%s\"}==1)", r.Config.GetNamespace())),
+			For:    "60s",
+			Labels: map[string]string{"severity": "critical"},
+		}, {
+			Alert: fmt.Sprintf("AMQOnlineKeycloakUnavailable"),
+			Annotations: map[string]string{
+				"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md",
+				"message": fmt.Sprintf("AMQ-SLI-1.3: Keycloak is not available in namespace %s", r.Config.GetNamespace()),
+			},
+			Expr:   intstr.FromString(fmt.Sprintf("absent(kube_endpoint_address_available{endpoint=\"standard-authservice\",namespace=\"%s\"}==\"%s\")", r.Config.GetNamespace(), keycloakServicePortCount)),
+			For:    "60s",
+			Labels: map[string]string{"severity": "critical"},
+		}, {
+			Alert: fmt.Sprintf("AMQOnlineAddressSpace"),
```

Comment:
This should be about the ability to create AddressSpaces & Addresses, which is based on the enmasse-operator being available.
Maybe `AMQOnlineOperatorAvailable` is a better name